Detectors/FOCAL/simulation/data/simcuts.dat

```diff
@@ -0,0 +1,16 @@
+* FOCAL  
```

Comment:
@sawenzel How do we treat detector-specific simcuts? Can they stay in the FOCAL directory or should they be merged with the global simcuts?